src/toolbar_gui.cpp

```diff
@@ -1150,8 +1150,6 @@ void SetStartingYear(TimerGameCalendar::Year year)
 {
 	_settings_game.game_creation.starting_year = Clamp(year, CalendarTime::MIN_YEAR, CalendarTime::MAX_YEAR);
 	TimerGameCalendar::Date new_date = TimerGameCalendar::ConvertYMDToDate(_settings_game.game_creation.starting_year, 0, 1);
-	/* If you open a savegame as scenario there may already be link graphs.*/
-	LinkGraphSchedule::instance.ShiftDates(new_date - TimerGameCalendar::date);
```

Comment:
Why this removal?

Reply:
This is in `SetStartingYear()`, which changes TimerGameCalendar::date and used to shift linkgraph dates. But linkgraphs are now in TimerGameEconomy, so they don't care about the calendar date.

Reply:
But in this PR they are kept in sync, right?